lms/djangoapps/certificates/tests/test_support_views.py

```diff
@@ -339,6 +371,36 @@ def _regenerate(self, course_key=None, username=None):
 
         return self.client.post(url, params)
 
+    def _invalidate_certificate(self, certificate):
+        """ Dry method to mark certificate as invalid. """
+        CertificateInvalidationFactory.create(
+            generated_certificate=certificate,
+            invalidated_by=self.support,
+            active=True
+        )
+        # Invalidate user certificate
+        certificate.invalidate()
+        self.assertFalse(certificate.is_valid())
+
+    def assertInvalidatedCertExists(self, exists):
```

Comment:
I think this would be more readable if you had separate functions: `assertInvalidatedCertExists` and `assertInvalidatedCertDoesNotExist`